.openpublishing.redirection.fundamentals.json

```diff
@@ -568,6 +568,10 @@
         {
             "source_path_from_root": "/docs/fundamentals/networking/http/http-autoclient.md",
             "redirect_url": "/dotnet/fundamentals/networking/http/http-overview"
+        },
+        {
+            "source_path_from_root": "/docs/fundamentals/syslib-diagnostics/syslib5003.md",
+            "redirect_url": "/dotnet/fundamentals/syslib-diagnostics/syslib5003"
```

Comment:
@gewarren Do we need this? I don't understand the logic for which pages need this and which don't.

Reply:
No, this is only needed if you renamed or deleted a file after it was published.

Reply:
Thanks; this file can be reverted then.